recipes/openblas/bld.bat

```diff
@@ -0,0 +1,11 @@
+
+:: Set $HOME to the current dir so msys runs here
+set HOME=%cd%
+
+:: Configure, build, test, and install using `make`.
+bash -lc "make FC=gfortran DYNAMIC_ARCH=1 BINARY=$ARCH NO_LAPACK=0 NO_AFFINITY=1 USE_THREAD=1 PREFIX=`echo $LIBRARY_PREFIX | tr '\\' '/'`"
```

Comment:
Ah ok, thanks for clarifying. I have basically started learning about Windows because of this project and a few users of mine that would be a little happier having native Windows support. So, if I say something stupid, please correct me. :smile: